Umbraco-Cloud/Release-Notes/2022-03-29-ReleaseNotes.md

```diff
@@ -0,0 +1,15 @@
+# Release Notes, March 29, 2022
+
+_Bandwidth Usage Top 10 for resources_
+
+### Key Takeaways
+- **Bandwidth Usage Top 10 for resources** - Wondering which resources of your Umbraco Cloud project that contributes with most bandwidth usage? Now you can find the answer in the new bandwidth Top 10 for resources on the project usage page.
+
+### Bandwidth Usage Top 10 for resources
+The bandwidth top 10 for referrers is now out of beta. And we are introducing a **new top 10 for resources** where you will see the resources that are contributing the most to the total bandwidth of your project. Each resource is represented by its path together with the number of requests and its total contribution of bandwidth.
+
+![Top10BandwidthPaths](https://user-images.githubusercontent.com/93588665/160573035-09f62ec3-9f4a-49c8-8e90-163c7418cb75.gif)
```

Comment:
@sofietoft  do we allow images to be hosted externally?

Reply:
No images should add them to the images folder :) 

It has been resolved now :D